components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -384,11 +384,9 @@ def enable_bs4_layout
 
     def resolve_layout
       case action_name
-      when "step", "review_and_submit", "eligibility_response_error", "application_publish_error", "preferences"
+      when "step", "eligibility_response_error", "application_publish_error", "preferences"
         "financial_assistance_progress"
```

Comment:
by the way, is this correct? seems like a recent PR merged into our feature branch sets these four actions to always use our new layout regardless of the BS4 flag

Reply:
This should be featured flagged, good catch